test/spec/modules/currency_spec.js

```diff
@@ -0,0 +1,248 @@
+
+import {
+  getCurrencyRates
+} from 'test/fixtures/fixtures';
+
+import {
+  setConfig,
+  addBidResponseDecorator,
+
+  currencySupportEnabled,
+  currencyRates
+} from 'modules/currency';
+
+var assert = require('chai').assert;
+var expect = require('chai').expect;
+
+describe('currency', function () {
+  describe('setConfig', () => {
+    it('results in currencySupportEnabled = false when currency not configured', () => {
+      setConfig({});
```

Comment:
These tests pollute the global state with side-effects. Tests should be side-effect-free.